.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - sudo apt-get update -qq
 - sudo apt-get install -qq
     alien bison++ bisonc++ cmake doxygen freeglut3-dev flex gettext
-    libc6 libc6-dev libcap-dev libfreetype6 libfreetype6-dev
+    libboost-dev libc6 libc6-dev libcap-dev libfreetype6 libfreetype6-dev
```

Comment:
Are we including boost C++ libraries for GEE now?

Reply:
Not that I'm complaining :).  Lots of good stuff in there.

Reply:
I explained this in the commit message, but it seems like some versions of the `libyaml-cpp-dev` don't list `libboost-dev` as a dependency, but `libyaml` uses Boost shared pointers, so the build fails, if `libboost-dev` isn't installed.

The `libyaml-cpp-dev` package on my Ubuntu 14 installation seems to list the dependency, but it failed for Travis automated build, so I added it here to fix the build.  We may be able to remove the explicit `libboost-dev` listing in the future.